Signal/src/call/CallService.swift

```diff
@@ -608,7 +610,7 @@ fileprivate let timeoutSeconds = 60
         call.state = .connected
 
         // We don't risk transmitting any media until the remote client has admitted to being connected.
-        peerConnectionClient.setAudioEnabled(enabled: true)
+        peerConnectionClient.setAudioEnabled(enabled: !call.isMuted)
```

Comment:
This is a good change. Did you test that this behaves as expected? E.g. the call is initially muted, but it's possible for the caller to un-mute?

The reason I wanted to double check was that there is some plumbing related to setting up the AudioSession in RTCAudioSession that occurs only once you `setAudioEnabled(enabled: true)`.